app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
```

Comment:
We could break down visit_cafe functionality into two separate methods that will checks for vaccine and checks for mask to follow single responsibility principle.